examples/platform/esp32/common/CHIPDeviceManager.cpp

```diff
@@ -83,7 +83,10 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
     ReturnErrorOnFailure(PlatformMgr().StartBackgroundEventLoopTask());
 
     // Start a task to run the CHIP Device event loop.
-    return PlatformMgr().StartEventLoopTask();
+    ReturnErrorOnFailure(PlatformMgr().StartEventLoopTask());
+
+    // This is a no op if CONFIG_ENABLE_BG_EVENT_PROCESSING is disabled
+    return PlatformMgr().StartBackgroundEventLoopTask();
```

Comment:
We have started the background event loop task at line 83.

Reply:
Thanks @wqx6 for noticing, removed!